### ISSUE TITLE
Collection Runner does not respect environment variables at URL level

### ISSUE THREAD

**Issue by cdasilvasonnysdirect** [a85/POSTMan-Chrome-Extension#8894]
### Description

Whenever you add an environment variable into the URL and execute the endpoint with the collection runner it does not use that variable at all and replaces it with nothing (null, empty, empty string, equivalent).

#### Example:

```
URL: www.mywebapplication.com
```

Create the environments and declare a variable called `env` in each:

#### Stage

```
env = stage.
```

#### QA

```
env = qa.
```

#### Production

```
env = www.
```

The URL with the variable will be:

```
URL: {{env}}mywebapplication.com
```

So in each of the environments it is expected to execute as:

* stage.mywebapplication.com
* qa.mywebapplication.com
* www.mywebapplication.com

Executing the endpoint individually, outside of the collection runner, the environment variable is correctly replaced. Executing the endpoint through the collection runner, with or without the whole collection selected, does not replace the environment variable.

### Expected results

While executing the collection through the collection runner it is expected that the URL has the environment variables correctly replaced.

**Comment by DannyDainton** [comment]
Hey @cdasilvasonnysdirect 

Could you confirm that all the requests in the collection are saved and that you're selecting the environment in the Collection Runner before the run?

You mentioned _replaces it with nothing (null, empty, empty string, equivalent)_ could you take a screenshot of the Collection Runner console for the run to see where the request is being sent, please?

**Comment by cdasilvasonnysdirect** [comment]
@DannyDainton I will replicate an environment similar to mine to screenshot this. Give me a few.

**Comment by mccannt** [comment]
@cdasilvasonnysdirect Can you also provide us with the version of the app and your OS?

**Comment by cdasilvasonnysdirect** [comment]
Postman Version: 7.28.0
O.S.: macOS Catalina 10.15.5

**Comment by cdasilvasonnysdirect** [comment]
@DannyDainton, @mccannt I apologize. I started having a different error with the new mocking collection where the URLs were completely empty. Checking this specific error I have found out that, unless I explicitly hit the save button, the newly applied changes are not saved. Returning to my original collection I was able to find out that none-of-the changes performed were saved leading me to these wrong results. Once that was repaired all is working as expected.
I'm closing this as it is obviously a non-issue.
